stablehlo/dialect/StablehloOps.td

```diff
@@ -1356,11 +1356,13 @@ def StableHLO_AllGatherOp : StableHLO_Op<"all_gather",
 
     Example:
     ```mlir
-    %result = "stablehlo.all_gather"(%operand) {
+    %result:2 = "stablehlo.all_gather"(%operand0, %operand1) {
       all_gather_dim = 1 : i64,
       replica_groups = dense<[[0, 1]]> : tensor<1x2xi64>,
+      // channel_id = 0
```

Comment:
Let's remove the default value (`channel_id` and `use_global_device_ids` from the td file).

Reply:
why ? 
from the spec too?

Reply:
Just some old decision made in 7.iii https://github.com/openxla/stablehlo/blob/main/docs/spec_checklist.md. FYI `use_global_device_ids` is a `UnitAttr` in MLIR, so should be written as `use_global_device_ids` (without the `=false`).

Reply:
Done, thanks for the reference.